scripts/ci/batch_lhci_leak.sh

```
@@ -0,0 +1,174 @@
+    
+    # Set Chrome path and flags for Lighthouse in DinD environment
+    export CHROME_PATH=/usr/bin/chromium-browser
+    export LIGHTHOUSE_CHROME_FLAGS="--no-sandbox --disable-dev-shm-usage --disable-extensions --disable-gpu --headless --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-renderer-backgrounding --disable-software-rasterizer --disable-setuid-sandbox --single-process --no-zygote --js-flags=--max-old-space-size=4096"
+    
+    # Use Makefile target for production setup, then run lighthouse with DinD-compatible flags
+    echo "🚀 Starting production services..."
+    make start-prod
+    
+    # Connect the DinD host to the website-network to allow Chrome to reach containers
+    echo "🌐 Connecting DinD host to Docker network..."
+    docker network connect website-network $(hostname) 2>/dev/null || echo "Already connected or connection not needed"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)